### PR TITLE
Add confirm modal in dapp browser, fix ui issue, delete useless code. Fix #493

### LINKED ISSUE #493: Dapp Browser: Add confirmation modal for personal_sign and transaction

### PR DESCRIPTION
**Pull request by AmazingBeerBelly**
Transaction Modal:

CleanShot 2020-09-26 at 00 44 10@2x

Personal Sign Modal:

CleanShot 2020-09-26 at 00 44 10@2x

Fix dapp warning modal issue:

CleanShot 2020-09-26 at 00 44 10@2x

### CODE REVIEW

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -542,7 +542,7 @@
     },
     "dappWarning": {
       "title": "Redirecionando para o Dapp de terceiros \"%{dappName}\"",
-      "body": "%{description}\n\nNota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o Termo de Acordo do Usuário e a Política de Privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e uso no Dapp.\n\n"
+      "body": "%{description}Nota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o Termo de Acordo do Usuário e a Política de Privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e uso no Dapp.\n\n"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:545]:
```suggestion
      "body": "%{description}Nota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o termo de acordo do usuário e a política de privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e usos no Dapp.\n\n"
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor changes on Portuguese Brazilian translation

**Review by chrisli30 — COMMENTED** [review]
Code looks good to me.

One minor change:
In the first screenshot, the "RBTC gas fee" should show only 6 decimal points; otherwise it's too long

**Comment by AmazingBeerBelly** [comment]
CleanShot 2020-09-26 at 00 44 10@2x

CleanShot 2020-09-26 at 00 44 10@2x

File: src/common/translations/pt-BR.json [hunk 2]
```diff
@@ -542,7 +542,8 @@
     },
     "dappWarning": {
       "title": "Redirecionando para o Dapp de terceiros \"%{dappName}\"",
-      "body": "%{description}\n\nNota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o Termo de Acordo do Usuário e a Política de Privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e uso no Dapp.\n\n"
+      "body": "%{description}\n\nNota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o termo de acordo do usuário e a política de privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e usos no Dapp.\n\n",
+      "defaultDescription": "Você está sujeito ao Acordo de Utilizador e Política de Privacidade de Dapp de terceiros quando entra no Dapp de terceiros, que é diretamente e separadamente responsável para você por \"%{dappName}\"."
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:546]:
```suggestion
      "defaultDescription": "Ao utilizar Dapps de terceiros, você está sujeito ao termo de uso e a política de privacidade desses Dapps, que neste caso são de responsabilidade e fornecidos por \"%{dappName}\"."
```

File: src/common/translations/pt-BR.json [hunk 3]
```diff
@@ -542,7 +542,8 @@
     },
     "dappWarning": {
       "title": "Redirecionando para o Dapp de terceiros \"%{dappName}\"",
-      "body": "%{description}\n\nNota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o Termo de Acordo do Usuário e a Política de Privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e uso no Dapp.\n\n"
+      "body": "%{description}\n\nNota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o termo de acordo do usuário e a política de privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e usos no Dapp.\n\n",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:545]:
```suggestion
      "body": "%{description}\n\nNota: Você será redirecionado para um Dapp de terceiros \"%{dappName}\". Por favor, analise cuidadosamente o termo de uso e a política de privacidade do Dapp. \"%{dappName}\" é responsável por todas as ações e usos no Dapp.\n\n",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor changes Portuguese Brazilian

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change

**Review by chrisli30 — APPROVED** [review]

**Review by imstar15 — APPROVED** [review]

**Pull request merged**